.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDurationParser.cs

```diff
@@ -267,9 +269,9 @@ private DateTimeResolutionResult ParseInexactNumberUnit(string text)
                     ret.FutureValue = ret.PastValue = double.Parse(numStr) * this.config.UnitValueMap[srcUnit];
                     ret.Success = true;
                 }
-                else if (match.Groups["business"].Success)
+                else if (match.Groups[Constants.BusinessDayGroupName].Success)
                 {
-                    ret.Timex = TimexUtility.GenerateDurationTimex(numVal, "BD", false);
+                    ret.Timex = TimexUtility.GenerateDurationTimex(numVal, Constants.TimexBusinessDay, false);
```

Comment:
This is still not ideal. The method parameters should be the info needed to generate a timex. The actual string value is an implementation detail internal to the method. The string itself should not be received as a parameter. Let's add this change to the refactor item too, ok?